### PR TITLE
[MAINT] security: bump OPA from 0.42.0 to 0.44.0

### PR DESCRIPTION
**Pull request by rafaeldtinoco**
This was triggered by: https://github.com/aquasecurity/tracee/pull/2169

And it is a maintenance PR.

Bump OPA from 0.42.0 to 0.44.0.
- [Release notes](https://github.com/open-policy-agent/opa/releases)
- [Changelog](https://github.com/open-policy-agent/opa/blob/main/CHANGELOG.md)
- [Commits](https://github.com/open-policy-agent/opa/compare/v0.42.0...v0.44.0)

### CODE REVIEW

**Review by josedonizetti — APPROVED** [review]

**Comment by rafaeldtinoco** [comment]
For the record:

https://github.com/aquasecurity/tracee/actions/runs/3079173636/jobs/4975198021

Our release is blocked until this is merge. Thanks for the +1.

**Pull request merged**
